server/container/guice/guice-common/src/main/java/org/apache/james/jmap/JMAPModule.java

```diff
@@ -85,19 +86,23 @@ private PropertiesConfiguration getConfiguration(FileSystem fileSystem) throws F
     }
 
     @Singleton
-    public static class MoveCapabilityPrecondition implements ConfigurationPerformer {
+    public static class RequiredCapabilitiesPrecondition implements ConfigurationPerformer {
 
         private final MailboxManager mailboxManager;
+        private final MessageManager messageManager;
 
         @Inject
-        public MoveCapabilityPrecondition(MailboxManager mailboxManager) {
+        public RequiredCapabilitiesPrecondition(MailboxManager mailboxManager, MessageManager messageManager) {
```

Comment:
Yes that was my point, I think we should keep capabilities in MailboxManager for now.